### DISCUSSION TITLE
Welcome to the Introduction to Bash Scripting eBook Discussions!

### DISCUSSION THREAD

**Discussion by bobbyiliev** [bobbyiliev/introduction-to-bash-scripting#75] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blackcrack** [comment]
Hi,

imho should all what here in Discussion posted or easy post some Code-parts here in Discussions, should be able to add in the "[introduction-to-bash-scripting](https://github.com/bobbyiliev/introduction-to-bash-scripting)" under the same Copyleft as it is this [introduction-to-bash-scripting](https://github.com/bobbyiliev/introduction-to-bash-scripting) for the World and any user avail who search informations for his does.. as help and "solution solver" or how ever \*g*.. , so all what here Posted as Code, should be able to pour in/make mention of this "[introduction-to-bash-scripting](https://github.com/bobbyiliev/introduction-to-bash-scripting)" as Head Rules for Posting in this Discussion without name of the User and Time, so anonym.

Who want be not agree, should not posting here a Comment.

all the mentions here gives other lessons and solutions for other projects.
And all Users who posting here, must be agree to help other in the world. therewith it is possible for collect many many informations for make this "[introduction-to-bash-scripting](https://github.com/bobbyiliev/introduction-to-bash-scripting)" an hughe Magic Book of Bash scripting :wink:

best regards
Blacky
Blackysgate.de

(maybe imho, close and sticky ? you are welcome to take over the text and add it to yours and delete this post )
